packages/mercury/src/components/dropdown/_dropdown-styles.scss

```diff
@@ -1,5 +1,11 @@
+@use "../../base/button/button-styles" as *;
 @use "../../base/common" as *;
 @use "../../base/typography" as *;
+@use "../../base/icons" as *;
+
+%dropdown {
+  display: contents; // Prevents unintended button styles from affecting the host element.
+}
```

Comment:
This selector won't have enough specificity of the dropdown bundle is downloaded before the button bundle, in other word, right now the specificity of the button selector and this selector is the same.